### DISCUSSION TITLE
Velero backup storage with Azure Storage Blob requires Service Principal with 'Contributor' access - any other options?

### DISCUSSION THREAD

**Discussion by sankalita-97** [vmware-tanzu/velero#7918] [Community support Q&A]
We have a velero backup dedicated storage blob in  Azure, that uses a Service Principal with 'Contributor' access for access from the AKS cluster. Is there any other permission level that we can degrade from the 'Contributor' access that would work with Velero?

**Comment by blackpiglet** [comment]
https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure?tab=readme-ov-file#specify-role

As this document specified, you can choose the existing `contributor` role, or create a customized role.
